src/LGPL-2.0+.xml

```diff
@@ -6,15 +6,6 @@
       <crossRefs>
          <crossRef>http://www.gnu.org/licenses/old-licenses/lgpl-2.0-standalone.html</crossRef>
       </crossRefs>
-      <standardLicenseHeader> Copyright (C) 
-    <alt match=".+" name="copyright">year name of author</alt> This library is free software; you can redistribute
-    it and/or modify it under the terms of the GNU Library General Public License as published by the Free Software
-    Foundation; version 2. This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
```

Comment:
This was missing the critical “any later version”.  This also affected `LGPL-2.1+`.